charts/backingservices/README.md

```diff
@@ -113,14 +113,14 @@ srs:
     #port: 9200
     #protocol: https
     # The elasticsearch connection supports three authentication methods: basic authentication , AWS IAM role-based authentication and Elasticsearch secure connection(tls).
-    # To configure tls on internal cluster uncomment and set `srs.srsStorage.tls.enabled: true`
+    # set `srs.srsStorage.tls.enabled: true` to enable tls on internal/external elasticsearch cluster.
     tls:
      enabled: false
     # Specify the certificate name used to connect to elasticsearch when tls.enabled: true and srsStorage.provisionInternalESCluster: false.
-    # certificateName: "elastic-certificates.p12"
-    # To configure basicAuthentication on internal cluster uncomment and set `srs.srsStorage.basicAuthentication.enabled: true`
+    # certificateName: "Certificate_Name"
+    # set `srs.srsStorage.basicAuthentication.enabled: true` to enable basicAuthentication on internal/external elasticsearch.
```

Comment:
Set `srs.srsStorage.basicAuthentication.enabled: true` to enable the use of basic authentication to your Elasticsearch service whether is it running as an internalized or externalized service in your SRS cluster.